others/index.adoc

```diff
@@ -5,6 +5,20 @@ Emmanuel Bernard
 We always have new ideas to solve data related problems.
 You will find some of these experiments here.
 
+== Hibernate OGM
+
+[role="ui message warning"]
+Hibernate OGM has not been kept up to date with the latest Hibernate ORM versions
```

Comment:
Shouldn't we add a banner with a similar message to all OGM pages? Otherwise, people coming from external websites won't know about it...